backend/src/main/java/dccs/academy/utils/mappers/CertificateMapper.java

```diff
@@ -12,7 +13,7 @@ public static CertificateDto toDto(CertificateEntity certificateEntity) {
         certificateDto.setId(certificateEntity.getId());
         certificateDto.setValidFrom(certificateEntity.getValidFrom());
         certificateDto.setValidTo(certificateEntity.getValidTo());
-        certificateDto.setType(certificateEntity.getType());
+        certificateDto.setType(CertificateType.stringToEnum(certificateEntity.getType().getDisplayName()));
```

Comment:
Oh no, in our world - code - this is enum and it is important to keep it enum in dto. In user world - GUI - this is some translated value. So, enum translations are added just as any other translation to your translations files.